src/components/controls/filter.js

```diff
@@ -85,10 +89,26 @@ class FilterData extends React.Component {
           });
         });
     }
+    if (this.props.measurementsOn && this.props.measurementsFiltersMap && this.props.measurementsFieldsMap) {
+      this.props.measurementsFiltersMap.forEach(({values}, filterField) => {
+        const { title } = this.props.measurementsFieldsMap.get(filterField);
+        values.forEach((value) => {
+          options.push({
+            _type: "measurements", // custom field to differentiate measurements filters
+            label: `${title} → ${value}`,
```

Comment:
I think we need some UI hint that these are for the measurement panel only, especially as the `title` is arbitrary and I can imagine collisions (e.g. tree nodes with "source" and measurement panel with "source"). Not a whole lot of visual space to play with here tho 😅

Reply:
Yeah, I think upgrading `react-select` to use option groups is the best thing to do here...

For now, would it be confusing for measurements options to be `(M) ${title} → ${value}`? I can add a note within `FilterInfo`.

Reply:
Yes I think that's an acceptable solution for now -- it's a good trade off between not adding characters (as overflowing labels are not correctly displayed) but indicating something is different. The user-base for measurements is expected to be close collaborators for at least the time before we upgrade react-select!